lib/Notifications.dart

```diff
@@ -0,0 +1,208 @@
+import 'dart:ui';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'Home.dart';
+
+class Notifications extends StatefulWidget {
```

Comment:
Make sure that you are able to navigate to the Notifications page from the menu on the homepage. I was unable to navigate there.